extensions/wikia/Paginator/Paginator.body.php

```diff
@@ -93,6 +93,12 @@ public function hasContent(){
 		return ( $this->pagesCount >= 1 );
 	}
 
+	/**
+	 * Set the currently active page. This is 0-indexed, so you may need to
+	 * set the value to $this->getRequest()->getInt( 'page' ) - 1
+	 *
+	 * @param $iPageNumber
```

Comment:
While there isn't an explicit rule against type based variable names,  in general the MW guidelines for PHP discourage the use of prefixes (except for `wg` for globals):

> https://www.mediawiki.org/wiki/Manual:Coding_conventions/PHP#Variables

You can note the type of this variable above by adding it before the variable name, e.g.:
```php
@param int $pageNumber
```